notebooks/pcts_census_utils.py

```diff
@@ -168,10 +169,12 @@ def income_percentiles(row, percentiles, prefix="total"):
         'WDI', 'WTM', 'YV', 'ZAA', 'ZAD', 'ZAI', 'ZBA', 'ZC', 'ZCJ', 'ZV', 
         ]
 
+
 # Subset PCTS and only get parent cases
-def subset_pcts_drop_child(start_date, prefix_list=None, suffix_list=None):
+def get_pcts_parents(start_date, end_date=None, prefix_list=None, suffix_list=None):

```

Comment:
Can we also allow `start_date` to be `None`? I'd like to be able to call this function with no args and get something sensible.

Reply:
Ok, it's optional, but I misspoke, I checked `src/A4_create_pcts_master.py`, and that one is already keeps 2010 as the earliest year. So here, `start_date` can be optional, with default being 1/1/2010. It still takes many minutes, not exactly sure how to use the %timeit, but it's so slowwww with the full prefix_list and suffix_list.

Reply:
Okay, cool, I'll poke around and see what I can do to speed things up. Do you have an idea of the slowest part?

Reply:
Possibly lines 240-250? I don't think it's the `drop_child_cases` function, unless working with lists is slow? I also just force-pushed to get rid of an extraneous comment I left for myself.